node/node_newblock.go

```diff
@@ -47,13 +47,20 @@ func (node *Node) WaitForConsensusReadyV2(readySignal chan struct{}, stopChan ch
 						Uint64("blockNum", node.Blockchain().CurrentBlock().NumberU64()+1).
 						Msg("PROPOSING NEW BLOCK ------------------------------------------------")
 
+					// Prepare last commit signatures
+					commitSigs := make(chan []byte)
+					sigs, err := node.Consensus.BlockCommitSigs(node.Blockchain().CurrentBlock().NumberU64())
```

Comment:
I am a little confused here: The sigs results are computed simultaneously. What's the intention of adding channel logic here? Oh, is it true that the pipeline feature is not implemented in this PR?

Reply:
Sorry, didn't clarify in the description. This pr is just adding the async part for the block proposal to wait on the commitSigs. This PR is not implementing the real early block proposal (pipelining) part yet. just want to make things separate.

Reply:
OK, sure, NP